package.json

```diff
@@ -20,6 +20,11 @@
   ],
   "license": "MIT",
   "main": "index.js",
+  "files": [
+    "base64js.min.js",
```

Comment:
Depending on whether #66 will be included before this line might have to be removed.